### PR TITLE
Fix CallKit integration on iOS for conferences requiring moderator

### PR DESCRIPTION
**Pull request by jleren**
When you join a conference that needs an authenticated moderator, as a guest, Jitsi Meet will continuously try and connect to the meeting every 5 seconds.
For every retry, on iOS, a new CallKit call is created, causing the call to loose audio, and loose control over the call.

This fixes that, by only creating a call on CallKit, if one has'nt already been created.

### CODE REVIEW

**Comment by jleren** [comment]
Should also fix #6260

**Comment by AliKarpuzoglu** [comment]
Amazing. I tested it out and it works for me. Both when opening a room as well as joining a lobby with a password.

**Comment by jleren** [comment]
The CCLA is now signed for my company, ThisIsUniverse ApS

**Comment by saghul** [comment]
Sorry for the delay, I've been AFK for the past 3-4 weeks. Thanks for the fix! Now, I think we should apply it to Android as well. As it would likely confuse ConnectionService. Can you please remove the platform check? Cheers!

File: react/features/mobile/call-integration/middleware.js [hunk 1]
```diff
@@ -264,6 +264,11 @@ function _conferenceWillJoin({ dispatch, getState }, next, action) {
     const handle = callHandle || url.toString();
     const hasVideo = !isVideoMutedByAudioOnly(state);
 
+    // If we already have a callUUID set, dont start a new call
```
Comment (saghul) [inline react/features/mobile/call-integration/middleware.js:267]:
```suggestion
    // If we already have a callUUID set, don't start a new call.
```

**Comment by saghul** [comment]
Tiny tiny nit in the comment, LGTM otherwise, thank you!

**Comment by jleren** [comment]
I havent been able to test the Android ConnectionService, as I currently dont have an android phone, but the change should not break anything.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Cheers!

**Pull request merged**